### ISSUE TITLE
New lua branch update giving weird results

### ISSUE THREAD

**Issue by dutchgecko** [lukas-reineke/indent-blankline.nvim#40]
I just updated to the lua branch after seeing your reddit post about the virtual text support (and built nvim as well, of course). However I'm seeing some odd behaviour straight away while viewing my init.vim.

![image](https://user-images.githubusercontent.com/2513484/108733474-d2c4f280-752e-11eb-9ff9-4c78ca6e7c41.png)

In the above screenshot, column 1 always has an indentline drawn in it, which is not what I'd expect. In the indented portions, other indents are sometimes *not* drawn.

Relevant settings:

```vim
set expandtab
set smarttab
set tabstop=4
set shiftwidth=4

let g:indentLine_char = ''
let g:indentLine_bufTypeExclude = ['help', 'terminal']
```

**Comment by lukas-reineke** [comment]
I'll add an option to turn off the indent in the first column #12 

As for the missing lines, I'm not too sure. When I open the file it looks fine.
Can you try to create a simple way to reproduce this?

![Screen Capture_select-area_20210223121404](https://user-images.githubusercontent.com/12900252/108798338-f2145c00-75d0-11eb-9741-33b34fee8682.png)

**Comment by dutchgecko** [comment]
I can't reproduce this any more either. Gonna assume it was a one-off thing. 

Thanks for the fix for #12, looks great!
